index.js

```diff
@@ -17,11 +18,33 @@ const action = async context => {
 	const split = context.config.get('path').split('/');
 	const bucket = split.shift();
 	const filename = path.basename(filePath);
+	const extension = path.extname(filename);
+	const objectKey = path.join(split.join('/'), filename);
+
+	let contentType = 'application/octet-stream';
```

Comment:
Can you extract these into a map like this

```js
const contentTypes = new Map([
    ['.gif', 'image/gif'],
    [...]
]);

const contentType = contentTypes.get(extension) || 'application/octet-stream';
```

Reply:
Nice use of Map!  Thanks for the suggestion.  Changed here: https://github.com/SamVerschueren/kap-s3/pull/3/commits/5fc247fa3e3cce1db531fc58e704a355f9adc117